msgs/msg/SubTrajectory.msg

```diff
@@ -0,0 +1,19 @@
+# unique id within task
+uint32 id
+
+# associated cost
+float32 cost
+
+# optional associated name of sub trajectory
+string name
```

Comment:
For example we want to name different grasp solutions.

Reply:
Hm, this could be considered as a special-case of the `any` properties of InterfaceStates.
We did not discuss how they fit into the message structure yet. I believe it makes a lot of sense to publish them for debugging why stages miss parameters or received wrong parameters..

If you prefer to have an additional name field here, that's ok with me.